pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/creator/impl/fwd/SingleValueVarByteRawIndexCreator.java

```diff
@@ -38,6 +38,7 @@
  */
 public class SingleValueVarByteRawIndexCreator implements ForwardIndexCreator {
   private static final int DEFAULT_NUM_DOCS_PER_CHUNK = 1000;
+  private static final int TARGET_MIN_CHUNK_SIZE = 4 * 1024;
```

Comment:
This lower bound is debatable. 4KB is what we tested with, but since it's uncompressed target size it could be below disk read ahead value for many systems